section2/reflection.md

```diff
@@ -2,14 +2,26 @@
 
 1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
 
+I like the SMART acronym. It will help me be able to break projects down into realistic chunks to work on.
+The reading also helped to reinforce that it is ok to ask for help.
+
 2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
 
+I've always had an issue asking for help, because I've always been expected to just know the answer, and been treated poorly for asking.
+I also need to realize that mistakes are part of the learning, and stop trying to be a perfectionist.
```

Comment:
It's a big change from the way a lot of us grew up and were conditioned about what "learning" is.